storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.3.rst

```diff
@@ -0,0 +1,37 @@
+============================
+ |Percona XtraBackup| 2.3.3
+============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.3 on December 17th 2015. Downloads are available from our download site `here <http://www.percona.com/downloads/XtraBackup/xtrabackup-2.3.3/>`_. Downloads are available from our `download site <http://www.percona.com/downloads/XtraBackup/LATEST/>`_ and from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3 series.
+
+Bugs fixed
+----------
+
+ Database directories were not removed if ``DROP DATABASE`` happened during the backup. Bug fixed :bug:`1461735`.
+
+ Backup would fails if :ref:`storing_history` feature was enabled. Bug fixed :bug:`1509812`.
+
+ In case of :option:`--defaults-extra-file` option was not the first command line argument |innobackupex| would fail to authenticate and backup would fail. Bug fixed :bug:`1511451`.
```

Comment:
xtrabackup / innobackupex now fails with descriptive error message if --defaults-extra-file is not specified first (i.e. specified second, third or Nth where N > 1)